src/lib/components/Swap/Summary/Summary.tsx

```diff
@@ -47,18 +51,21 @@ function TokenValue({ input, usdc, change }: TokenValueProps) {
 }
 
 interface SummaryProps {
-  input: Required<Pick<Input, 'token' | 'value'>> & Input
-  output: Required<Pick<Input, 'token' | 'value'>> & Input
+  input: CurrencyAmount<Currency>
+  output: CurrencyAmount<Currency>
   usdc?: boolean
 }
 
 export default function Summary({ input, output, usdc }: SummaryProps) {
+  const inputUSDCValue = useUSDCValue(input)
+  const outputUSDCValue = useUSDCValue(output)
+
   const change = useMemo(() => {
-    if (usdc && input.usdc && output.usdc) {
-      return output.usdc / input.usdc - 1
+    if (inputUSDCValue && outputUSDCValue) {
+      return parseFloat(inputUSDCValue.divide(outputUSDCValue).quotient.toString())
```

Comment:
This had 1 subtracted - does that need to stay in?

Reply:
ah yeah this is currently wrong for % change (if thats what we're trying to show)